core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/diskModels.scala

```diff
@@ -124,6 +124,10 @@ object FormattedBy extends Enum[FormattedBy] {
   final case object Cromwell extends FormattedBy {
     override def asString: String = "CROMWELL"
   }
+
+  final case object Hail extends FormattedBy {
```

Comment:
Adding new disk `FormattedBy` adds a bunch of boilerplate and is largely irrelevant on Azure. Do we need it?